joomla/joomla-cms
# [3.9] purge update cache only on user request

Pull Request for Issue #23663 .

### Summary of Changes
* Removed auto purge from model::findUpdates

Now we only purge if the user clicks the purge button in the backend.


### Testing Instructions
* Check for updates on the dashboard.
* Check for updates in the installer component.
* Test this pr with version 3.9.3 to see if 3.9.4 is detected
* Test this 3rd party components
* Simulate updates by manipulating xml updates files

Cache should update after 1 hour.


### Expected result
Doesn't check for updates in cache and updates after 1 hour


### Actual result
Always checks for updates.


### Documentation Changes Required
no
